------------------------------------------------------------------------
r14917 | michael | 2008-08-23 15:29:13 +0200 (Sa, 23. Aug 2008) | 3 lines

Increase buffer size to 16384 patch by Alexander E. Patrakov" patrakov gmail
This fixes a (probably not exploitable) buffer overflow (apparently unknown to its author).

------------------------------------------------------------------------
Index: libavcodec/dca.c
===================================================================
--- a/libavcodec/dca.c	(Revision 14916)
+++ b/libavcodec/dca.c	(Revision 14917)
@@ -69,7 +69,7 @@
 #define HEADER_SIZE 14
 #define CONVERT_BIAS 384
 
-#define DCA_MAX_FRAME_SIZE 16383
+#define DCA_MAX_FRAME_SIZE 16384
 
 /** Bit allocation */
 typedef struct {
